modules/reactor/include/utils/ReportingIDGeneratorUtils.h

```diff
@@ -16,6 +16,16 @@
 
 namespace ReportingIDGeneratorUtils
 {
+/**
+ * Enum item for reporting id assign types.
```

Comment:
please either point to clasd user documentation or describe each strategy again here